tcs-persistence/src/main/java/com/transformuk/hee/tis/tcs/service/model/Post.java

```diff
@@ -87,6 +87,8 @@
   private Set<Placement> placementHistory = new HashSet<>();
   @OneToMany(mappedBy = "post", fetch = FetchType.LAZY)
   private Set<PostTrust> associatedTrusts;
+  @OneToMany(mappedBy = "post", fetch = FetchType.LAZY, cascade = CascadeType.REMOVE)
```

Comment:
A `FetchType.LAZY` will mean that we need to make sure we fetch it for the Post load.  Good plan for the cascade type (I think).

Reply:
It's a good shout. I have changed it to the EAGER fetch type.